libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -259,9 +259,7 @@ float AP_MotorsMulticopter::get_current_limit_max_throttle()
     }
 
     // calculate the maximum current to prevent voltage sag below _batt_voltage_min
-    float batt_current_max = MIN(_batt_current_max, _batt_current + (_batt_voltage-_batt_voltage_min)/_batt_resistance);
```

Comment:
The alternative looks much better to me, but I wouldn't change it before Randy or Leonard look at this.